.gitignore

```diff
@@ -1,45 +1,87 @@
-node_modules/
-runner/
+# Copyright 2019 Google LLC
```

Comment:
Looks like this was added in https://github.com/google-github-actions/deploy-cloud-functions/pull/20/commits/3d1f1ef3620a2221ecf9665645f578cb0918b4d2
We should keep the existing gitignore and only append if needed.

Reply:
Sorry it was my bad. Fixed by reverting it back.

Reply:
Please remove these changes